extensions/wikia/PaidAssetDrop/js/PaidAssetDrop.desktop.js

```diff
@@ -0,0 +1,138 @@
+define('ext.wikia.PaidAssetDrop', [
```

Comment:
So, `ext.wikia.paidAssetDrop` or `ext.wikia.PaidAssetDrop.paidAssetDrop`? I'm not sure how to name the module... one of the biggest issues in programming ;)